events/hfe_events_init.txt

```diff
@@ -1055,9 +1108,18 @@ country_event = {
 					initializer = hfe_initializer_future_main
 				}
 				set_global_flag = hfe_future_spawned
+				if = {
+					limit = { has_global_flag = HFE_TSP_spawn_test }
+					log = "Kad Republic not found but clay spawned"
+				}
 			}
 		}
 		event_target:hfe_future_country = {
+			hfe_effect_init_give_tech_t5 = yes
```

Comment:
redundant

Reply:
It's a catch-all. It's not like you can have the same technology twice.

Reply:
Still this is not needed cpu load